### PR TITLE
Create arduino

### PR DESCRIPTION
**Pull request by ghost**
Create arduino code

### CODE REVIEW

File: src/main/arduino [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+void setup() {
+  Serial.begin(9600);
+}
+
+void loop(){
+
+    if (Serial.available()>0){
+        char data = Serial.read();
+
+        if(data != '\0'){
```
Comment (budougumi0617) [inline src/main/arduino]: If data length is bigger than maxlen, I think it occurs illegal memory access in buff, am I wrong?

File: src/main/arduino [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#include <LiquidCrystal.h>
+#define BUFF_LEN 64
```
Comment (budougumi0617) [inline src/main/arduino:2]: This variable name does not show the fact, because real buf length is 65.

File: src/main/arduino [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+LiquidCrystal lcd = LiquidCrystal(5, 6, 7, 8, 9, 10, 11);
+int counter = 0;
+int i;
+char buff[BUFF_LEN + 1];
```
Comment (budougumi0617) [inline src/main/arduino:7]: I think "+1" does not need by reason in above described comment.

File: src/main/arduino [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+void setup() {
+  Serial.begin(9600);
+}
+
+void loop(){
+
+    if (Serial.available()>0){
+        char data = Serial.read();
+
+        if(data != '\0'){
```
Comment (budougumi0617) [inline src/main/arduino:18]: Must add one more condition about buff length.
HINT: `buff` should not receive character greater than `BUFF_LEN`.

**Review by budougumi0617 — CHANGES_REQUESTED** [review]

**Comment by budougumi0617** [comment]
@ema195y Please check above my comments 😎

File: src/main/arduino (hunk 4 above)
Comment (ghost) [reply src/main/arduino:18]: https://www.arduino.cc/en/Serial/Available

**Review by budougumi0617 — APPROVED** [review]

**Pull request merged**